Lib/test/test_syntax.py

```diff
@@ -988,7 +988,7 @@ def _check_error(self, code, errtext,
     def test_expression_with_assignment(self):
         self._check_error(
             "print(end1 + end2 = ' ')",
-            'expression cannot contain assignment, perhaps you meant "=="?',
+            "cannot assign to expression. Maybe you meant '==' instead of '='?",
```

Comment:
Also dubious. Probably some other token was omitted.

Reply:
I mostly agree (although I think this has also a good chance of being the actual problem given that `x + y == z` could be mispelled easily) but in this case we don't add or remove existing functionality (the previous error was already along these lines) so we can try to address this in the future.